### ISSUE TITLE
ENCRYPT: Encrypt PDFs before upload

### ISSUE THREAD

**Issue by atangster** [anusii/healthpod#9]
# Description
Add encryption functionality to secure PDFs before they are uploaded to the POD.

# Why
To protect sensitive medical information during storage and transit.

**Comment by gjwgit** [comment]
Check if this is done by default.

**Comment by atangster** [comment]
Yes, this is done by default in readPod` and `writePod` functions from `solidPod` package.

- `readPod`: https://github.com/anusii/solidpod/blob/dev/lib/src/solid/read_pod.dart
- `writePod`: https://github.com/anusii/solidpod/blob/dev/lib/src/solid/write_pod.dart

But unclear if currently being done for large files (like PDFs) in demopod as it is using the large file helpers such as `sendLargeFile`, `getLargeFile` and `deleteLargeFile` which refer to `createResource` and `getResource`, but not `readPod` and `writePod`. Currently `createResource` and `getResource` do not have built-in encryption. 

- large file helpers: https://github.com/anusii/solidpod/blob/dev/lib/src/solid/utils/large_file_helper.dart
- demopod use of large file helpers: https://github.com/anusii/solidpod/blob/dev/demopod/lib/home.dart
